plugins/graphql/src/service/router.ts

```diff
@@ -29,17 +33,38 @@ const schemaPath = path.resolve(
 
 export interface RouterOptions {
   logger: Logger;
+  config: Config;
 }
 
 export async function createRouter(
   options: RouterOptions,
 ): Promise<express.Router> {
   const typeDefs = await fs.promises.readFile(schemaPath, 'utf-8');
 
-  const server = new ApolloServer({ typeDefs, logger: options.logger });
+  const catalogModule = await createCatalogModule(options);
+
+  const { schema } = new GraphQLModule({
+    imports: [catalogModule],
+    typeDefs,
+  });
+
+  const server = new ApolloServer({
+    schema,
+    logger: options.logger,
+  });
+
   const router = Router();
 
   const apolloMiddlware = server.getMiddleware({ path: '/' });
+
+  router.use(
+    helmet.contentSecurityPolicy({
+      directives: {
+        defaultSrc: ["'self'", "'unsafe-inline'", 'http://*'],
```

Comment:
This is needed to get `graphql-playground` playing nice with the latest versions of Chrome and the CSP policies with `helmet`

Reply:
Not great, any other options? Could we for example just stick to GraphiQL and skip serving the playground?